### ISSUE TITLE
Crome128 standardized CSS zoom property that leads to _getCoordinates return wrong result 

### ISSUE THREAD

**Issue by y122972** [ONLYOFFICE/DocumentServer#2859]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DocumentServer/issues?q=) and did not find an issue describing my bug.

### Operating System of DocumentServer

Windows Server

### Version information

latest

### Expected Behavior

right selection

### Actual Behavior

wrong selection

### Reproduction Steps

1. Change browser's zoom factor to 80%
2. Select some cell
3. Then we can see the wrong selection like this:

https://github.com/user-attachments/assets/19c5b04a-974c-4fa9-8fd4-6323bff5c238

### Additional information

Chrome 128 update release note: https://developer.chrome.com/release-notes/128#standardized_css_zoom_property

**Comment by yzxcxya** [comment]
> ### This issue is unique.
> * [x]  I have used the [search tool](https://github.com/ONLYOFFICE/DocumentServer/issues?q=) and did not find an issue describing my bug.
> 
> ### Operating System of DocumentServer
> Windows Server
> 
> ### Version information
> latest
> 
> ### Expected Behavior
> right selection
> 
> ### Actual Behavior
> wrong selection
> 
> ### Reproduction Steps
> 1. Change browser's zoom factor to 80%
> 2. Select some cell
> 3. Then we can see the wrong selection like this:
> 
>  aug10.mp4 
> ### Additional information
> Chrome 128 update release note: https://developer.chrome.com/release-notes/128#standardized_css_zoom_property

Hello, is there a temporary solution for this issue? Our customers are experiencing it as well.

**Comment by systemx-xx** [comment]
```
function getZoomLevel() {
	// Calculate the current zoom level as a percentage
	return Math.round((window.outerWidth / window.innerWidth) * 100);
}

function calculateRatio(zoomLevel, b = 100) {
	// Calculate the corresponding value of x based on the zoom level (a)
	const x = (b * b) / zoomLevel;
	return x;
}

const userAgent = navigator.userAgent;
const chromeVersion = userAgent.match(/Chrom(?:e|ium)\/([0-9]+)\./);

window.addEventListener('resize', () => {
	
	if (chromeVersion && parseInt(chromeVersion[1]) >= 127 ) {
		const x = calculateRatio(getZoomLevel());

		const searchString = "documenteditor";
		const iframes = document.querySelectorAll('iframe');
		const matchingIframes = Array.from(iframes).filter(iframe => iframe.src.includes(searchString));

		matchingIframes.forEach(iframe => {
			iframe.style.zoom = x + '%';
		});

	}
})
```

```
// After invoking the editor (in the event browser zoom is already zoomed during load
window.dispatchEvent( new Event('resize') );
```

**Comment by Rita-Bubnova** [comment]
Hello, thank you for report.
I can confirm - this is bug, issue #69884.

This has been fixed (https://github.com/ONLYOFFICE/sdkjs/commit/10706dfc3feace2473f5211b53c9ee1f13c7713f, https://github.com/ONLYOFFICE/sdkjs/commit/e35d5d8ed2a8dff7d876a79a889151dea2801628) and will be released in the version 8.1.3.

 I close this issue. Feel free to comment or reopen it if you got further questions.
